layouts/pieces/single.html

```diff
@@ -35,7 +35,7 @@ <h1>{{.Params.title}}</h1>
     <div class="footnotes-container"></div>
     <!-- See baseof.html for mobile-footnotes-container -->
 
-    {{ if isset .Params "footnotesurl" }}
+    {{ if isset .Params "footnotesUrl" }}
```

Comment:
You sure about this one? I vaguely remember the weird casing being a hugo quirk.

Reply:
It wouldn't compile otherwise

Reply:
> It wouldn't compile otherwise.

Are you sure? Footnotes are broken unless it's lower case. It's a hugo weirdness. See very bottom of https://gohugo.io/functions/isset/#readout. I reverted and pushed.